autonomous/autonomous.py

```diff
@@ -50,7 +50,7 @@ def __init__(self):
         # how far around the current position is used to smooth the path
         self.look_around = 0.3
 
-        print(f"[{self.MODE_NAME}] total length {self.total_length}s")
+        self.logger.info(f"[{self.MODE_NAME}] total length {self.total_length}s")
```

Comment:
Isn't MODE_NAME redundant here?
```suggestion
        self.logger.info(f"total length {self.total_length}s")
```

Reply:
all auto's get initialized so if you wanted to log their lengths you'd want a way to distinguish then, however we probably don't need to know the lengths

Reply:
:thinking: doesn't the logger show your mode's name already?

Reply:
yes, hadnt looked since it was changed from print to logger